src/Rector/Contrib/PHPUnit/SpecificMethod/AssertCompareToSpecificMethodRector.php

```diff
@@ -35,27 +36,41 @@
         'gettype' => ['assertInternalType', 'assertNotInternalType'],
     ];
 
+    /**
+     * @var string[][]|string[]
+     */
+    private $acceptedAssertionMethods = [
```

Comment:
@TomasVotruba Almost all the `Rector`s now has this array. It will be easier to extends functionality.